docs/src/examples/walt/animation.walt

```diff
@@ -22,7 +22,7 @@ let width: i32 = 0;
 let height: i32 = 0;
 export function onInit(totalParticles: i32): void {
   width = 500;
-  height = 300;
+  height = 400;
```

Comment:
Thanks, this fixes the problem and we can do better.

`getCanvasWidth` is imported but not used, because there is an error in the script. `GetSizeType` is undefined. Rather than hardcoding it we should call the utility function as it was meant to be part of the demo 🙃